.circleci/config.yml

```diff
@@ -243,7 +243,6 @@ workflows:
                 - /^Version-v(\d+)[.](\d+)[.](\d+)/
           requires:
             - prep-build
-            - get-changed-files-with-git-diff
```

Comment:
We can also remove `get-changed-files-with-git-diff`  as a requirement for the `test-e2e-firefox-flask` and `test-e2e-firefox-confirmation-redesign` jobs, which also only run on develop/master/rc

Reply:
that's a great point! Removed too :+1: